Website/Package.swift

```diff
@@ -0,0 +1,28 @@
+// swift-tools-version: 5.9
```

Comment:
```suggestion
// swift-tools-version: 6.0
```

Reply:
MyLibrary Package’s Swift 6 support was considered, but the impact was too large, and it was determined to be beyond the scope of this pull request.
Therefore, I reverted the version of the Website Package to match MyLibrary Package’s swift-tools-version to ensure compatibility and prevent potential build issues. 🙇‍♂️ https://github.com/tryswift/trySwiftTokyoApp/pull/57/commits/03645aec73df5ee24a09f2a6d1f01c0046e4614d